zerver/data_import/mattermost.py

```diff
@@ -1046,7 +1048,7 @@ def do_convert_data(mattermost_data_dir: str, output_dir: str, masking_content:
 
         # Export message attachments
         attachment: dict[str, list[Any]] = {"zerver_attachment": zerver_attachment}
-        create_converted_data_files(uploads_list, realm_output_dir, "/uploads/records.json")
         create_converted_data_files(attachment, realm_output_dir, "/attachment.json")
+        create_converted_data_files(uploads_list, realm_output_dir, "/uploads/records.json")
```

Comment:
Is this reordering intentional?  If so, why?

Reply:
Seems like it can't possibly have a functional effect. But it does place the `attachment` table closer to where it's used, so is probably positive. Will merge with this as-is.